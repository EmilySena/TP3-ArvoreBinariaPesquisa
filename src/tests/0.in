i bhuvis
i acrein
i ut
i ausklhein
i zhitan
i nhe
i baiean
i skhuls
e 4 skhuls ut nhe acrein
d 5 1 2 3 5 7
s skhuls aarain
e 4 aarain ut nhe acrein
