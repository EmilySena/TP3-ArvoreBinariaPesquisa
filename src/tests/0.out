7 5 6 2
bhuvis acrein ausklhein ut skhuls
3 6 7 2
